nametag_server.py

```diff
@@ -20,19 +20,19 @@ def allowed_file(filename):
     return '.' in filename and \
            filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
 
-def save_user_info(name, email):
+def save_user_info(name, pronoun, email):
     needs_header = False
     csv_file = os.path.join(os.sep, get_script_path(), CSV_FILENAME)
     logger.info("using CSV file at {}".format(csv_file))
     if not os.path.exists(csv_file):
         needs_header = True
 
     with open(csv_file, 'a') as csvfile:
-        fieldnames = ["Name", "Email", "Timestamp"]
+        fieldnames = ["Name", "Pronoun", "Email", "Timestamp"]
         writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
         if needs_header:
             writer.writeheader()
-        writer.writerow({"Name":name, "Email":email, "Timestamp":datetime.datetime.now()})
+        writer.writerow({"Name":name, "Preferred Pronoun":pronoun, "Email":email, "Timestamp":datetime.datetime.now()})
```

Comment:
Please make this key match with the `fieldnames` key: `"Pronoun"`, not `"Preferred Pronoun"`